backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -152,3 +152,13 @@ enum class CellStatusStyle {
      */
     Label,
 }
+
+private fun findBestWeekdayStyleForLocale(locale: Locale): TextStyle =
+    when (locale.language.lowercase()) {
+        "zh" -> when (locale.country.lowercase()) {
+            "mo", "hk", "tw", "sg" -> TextStyle.SHORT // Traditional
```

Comment:
can you also add a comment why we're doing this here?

Reply:
In addition to what we done in Kotlin Docs?

Reply:
yeah - kotlin docs is for users modifying the format externally, but here for whoever reads the code and wonders why this is there

Reply:
Done